### ISSUE TITLE
exec plugin: invalid apiVersion "client.authentication.k8s.io/v1alpha1"

### ISSUE THREAD

**Issue by tekumara** [loft-sh/vcluster#551]
### What happened?

```
$vcluster connect vcluster-1 -n playlab
fatal  find vcluster: create kube client: exec plugin: invalid apiVersion "client.authentication.k8s.io/v1alpha1"
```

### What did you expect to happen?

No error

### How can we reproduce it (as minimally and precisely as possible)?

Use same versions of EKS and vcluster

### Host cluster Kubernetes version

<details>

EKS v1.20

</details>

### Host cluster Kubernetes distribution

<details>

```
EKS
```

</details>

### vlcuster version

<details>

```console
$ vcluster --version
0.10.1
```

</details>

### Vcluster Kubernetes distribution(k3s(default)), k8s, k0s)

<details>

```
k3s
```

</details>

### OS and Arch

<details>

```
OS: darwin
Arch: amd64
```
</details>

**Comment by tekumara** [comment]
kubectl 0.24 ([introduced here in 0.90](https://github.com/loft-sh/vcluster/commit/a0716b6bddd490121d9acac13c90eefd673b8f45#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R40)) is not compatible with EKS 1.20.

Downgrading to vcluster 0.8.1 works.

**Comment by antonioberben** [comment]
This issue remains in vcluster `0.10.2`. Can anybody offer a workaround? Downgrading vcluster cli to `0.8.1` makes us to not enjoy the new features

**Comment by tekumara** [comment]
Thanks for the tip @antonioberben updating kubeconfig like this:
```
sed -i.bak 's/v1alpha1/v1beta1/g' ~/.kube/config
```
solved it for me, and works with vcluster 0.10.2.

**Comment by tekumara** [comment]
Thanks for the tip @antonioberben. [aws-cli moved to v1beta1](https://github.com/aws/aws-cli/blob/v2/CHANGELOG.rst#L163) in v2.7.0 (14 May) so upgrading aws-cli and updating kubeconfig like this:
```
sed -i.bak 's/v1alpha1/v1beta1/g' ~/.kube/config
```
solved it for me, and works with vcluster 0.10.2. 🥳
